frontend/pages/playscreen.vue

```diff
@@ -0,0 +1,465 @@
+<template>
+  <div class="playscreen">
+    <!-- Nur anzeigen, wenn isTrisightMode true ist -->
+    <div class="score-timer" v-if="isTrisightMode && !isGameOver">
+      <div class="left-side">
+        <!-- Anzeige des Game Timers als numerische Zeitangabe -->
+        <div class="game-timer">
+          Verbleibende Zeit: {{ gameTimerDisplay }} Sekunden
+        </div>
+        <div class="score">Punkte: {{ score }}</div>
+      </div>
+      <!-- Visuelle Darstellung des Timers als Kreis -->
+      <div class="timer-wrapper">
```

Comment:
I like the circle but I didn't notice it in the beginning. Same for the time. Just as an idea: maybe we can make them bigger/in a more central location?